File: samples/multi-project-plugin/common/src/main/java/example/common/ExampleBuildFeature.java

package example.common;

import jetbrains.buildServer.util.StringUtil;

public class ExampleBuildFeature {

    public static final String BUILD_FEATURE_NAME = "example";

    {
        // Just simple usage of TeamCity Common API (shared between server and agent)
        assert StringUtil.isTrue("true");
    }
}

File: samples/kotlin-plugin/common/src/main/kotlin/example/common/ExampleBuildFeature.kt

package example.common

import jetbrains.buildServer.util.StringUtil

object ExampleBuildFeature {

    const val BUILD_FEATURE_NAME = "example"

    init {
        // Just simple usage of TeamCity Common API (shared between server and agent)
        assert(StringUtil.isTrue("true"))
    }
}
